### PR TITLE
Spanish translation started

### PR DESCRIPTION
**Pull request by ijaureguialzo**
I've started the translation of iPeer into Spanish:

- Locale selector added. Now `display.locale` system parameter controls the whole application's language. It can be set to `spa` for Spanish or `eng` for English. Any other value defaults to English.
- Some controller titles reworked with `__()` calls added.
- New `.pot` file extracted with `cake i18n` command.
- Initial `.po` file with 1043 strings translated 100% into `es_ES` locale, using Poedit.
- Deleted English `.po` file, it's redundant.

### CODE REVIEW

**Comment by xcompass** [comment]
@ijaureguialzo Thanks for the PR. Could you please update the commit message into English? Most of us can't read Spanish unfortunately.  I'll take a look at the code changes, may have a few more comments.

File: app/views/home/index.ctp [hunk 1]
```diff
@@ -31,7 +31,7 @@ if (isset($course_list['A']))
           $eventReview = '<br />('.sprintf(__('%d unreviewed group evaluations', true), $event['to_review_count']).')';
       }
       $eventRatio = $event['completed_count'] .' of '.
-        $event['student_count'] . ' Students';
+        $event['student_count'] . __(' Students',true);
```
Comment (xcompass) [inline app/views/home/index.ctp]: You may want to change this into something like 
```php
$eventRatio = printf(__('%d of %d Students', true), $event['completed_count'], $event['student_count']);
```

**Comment by xcompass** [comment]
@ijaureguialzo it looks pretty good. :+1: Once you updated based on my comment above, could you squash the commits into one and use English as the description. Then I'll merge this PR. Thanks!

**Comment by ijaureguialzo** [comment]
Hi @xcompass, I'm unable to put the commit pack into a single commit, I haven't done that before. Could you help me with that?

I did the code change already, only the packing thing is pending.

**Comment by xcompass** [comment]
Here is what you need to do:
```
git checkout master
# this will give you a interactive interface, e.g. in VIM or nano to allow you pick which commit you want to squash
git rebase -i 47086d6debbc3e56b6a81cbcb5e35b42209375f1
# in the editor change the word pick to s from the second line, it should look like
pick xxxxxx xxxx
s xxxx xxxx
s xxxx xxxx
...
# then save and exit. git will open the editor again to allow you update commit message. This is where you can combine the message and summarize it. The preferred format is: first line is a short description within 72 characters. followed by one empty line and then a full description with multiple lines.
# once you finish, you can save and exit. Git will combine all commits into a single one. You can run:
git log
# to see the updated commits. Once satisfied, you need to force push (because you already pushed to github)
git push -f 
```
Hope it helps.

If there is any new commit in our master branch since your fork, you will need to do a rebase as well. Here is a good tutorial about rebase and squash: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

More info about squash commits: 
https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by ijaureguialzo** [comment]
Done. Something new learnt today. 

:)

**Comment by xcompass** [comment]
Thanks for the contribution!

**Pull request merged**